pytest_watcher/watcher.py

```diff
@@ -35,11 +42,41 @@ def emit_trigger():
         trigger = datetime.now()
 
 
-def _is_path_watched(filepath: str) -> bool:
-    """
-    Check if file should trigger pytest run
-    """
-    return filepath.endswith(".py")
+class FileFilter:
```

Comment:
Do you think we really need a class here? Can't we just use a plain function and store the state in a global variable(s)?